Azure-Samples/Microchip-PIC-IoT-Wx
# Reduce logging to allow v1.6.1 compiler binary to fit in flash

## Purpose
Reduce memory footprint to counter the 2% increase in binary size when moving from compiler v1.5.0 to v1.6.1

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Installed compiler 1.6.1
* Run through the readme.md, make sure 1.6.1 compiler is selected.

## What to Check
Make sure the application builds and fits into available flash

## Other Information
NA
